_security-analytics/security.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: default
+title: OpenSearch Security for Security Analytics
+nav_order: 2
+has_children: false
+---
+
+# OpenSearch Security for Security Analytics
+
+You can use OpenSearch Security with Security Analytics to assign user permissions and manage the actions that users can and cannot perform. For example, you might want one group of users to be able to create, update, or delete detectors and another group of users to only view detectors.
+
+All Security Analytics indexes are protected as system indexes. Only a super admin user or an admin user with a TLS certificate can access system indexes. For more information, see [System indexes]({{site.url}}{{site.baseurl}}/security/configuration/system-indices/).
```

Comment:
Is a super admin a different concept from the user with an admin certificate? I thought you had to have the admin cert to modify system indexes. If that's not the case can you define what a "super admin user" is, or link to a place in the docs that defines it?